bin/pingdom

```diff
@@ -36,12 +36,12 @@ save-check --template <template> --template-args key=value ...
   Process a template and save it, creating a new one or updating an existing entry by the same check name.
 
 Example:
-$0 save-check \
-  --template fhir-resource \
-  -a name=production-dstu2-medication-statement \
-  -a host=api.va.gov \
-  -a url="/services/fhir/v0/dstu2/MedicationStatement?patient=1011537977V693883" \
-  -a authorization_token="\$ARGONAUT_TOKEN" \
+$0 save-check \\
```

Comment:
Should we have other examples for checks that arent `save-check`.

Reply:
nah ...